libraries/AP_BattMonitor/AP_BattMonitor_SMBus.cpp

```diff
@@ -147,6 +215,50 @@ bool AP_BattMonitor_SMBus::read_word(uint8_t reg, uint16_t& data) const
     return true;
 }
 
+// read_block - returns number of characters read if successful, zero if unsuccessful
```

Comment:
Can you say that this method was moved up unmodified?

Reply:
I needed to use the method for the device and manufacturer functions. So either move the method up or the functions down. Wasn't sure the best way to deal with it? 

There are 3-4 versions of the `AP_BattMonitor_SMBus::read_block` floating around in some of the SMBus derived classes. They are analogs that could likely just use this method instead.

Reply:
@hendjoshsr71 sorry, I wasn't clear in my original question.

I wasn't keen on making sure this was just byte-for-byte moved without modification from the subclass and was seeking assurance that it was just moved.

Reply:
Seems like I forgot to answer this one. Yes it was just a copy & paste. I'm not sure why the git changes for this function look so weird where I removed it.